File: abc034/src/b.go
package main

import (
	"fmt"
)

func main() {
    var n int
    fmt.Scan(&n)
    ans := n
    if n % 2 == 0 {
        ans -= 1
    } else {
        ans += 1
    }
    fmt.Println(ans)
}

File: abc034/src/b.cpp

#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin >> n;

    int ans = (n % 2 == 0) ? (n - 1) : (n + 1);
    cout << ans << endl;
    return 0;
}